Source/RapyutaSimulationPlugins/Public/Core/RRUObjectUtils.h

```diff
@@ -65,11 +69,11 @@ class RAPYUTASIMULATIONPLUGINS_API URRUObjectUtils : public UBlueprintFunctionLi
 
     /**
      * @brief Return true if InParentObj has subobjects with InChildName.
-     * 
-     * @tparam T 
-     * @param InParentObj 
-     * @param InChildName 
-     * @return static bool 
+     *
+     * @tparam T
```

Comment:
please remove t at beginning

Reply:
Is that meant for template type, like in this UE [class](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Plugins/Animation/RigLogic/Source/RigLogicLib/Public/dna/types/ArrayView.h#L46)?